app-accessibility/espeak/espeak-1.48.04.recipe

```diff
@@ -35,22 +35,27 @@ BUILD_PREREQUIRES="
 	cmd:make
 	cmd:gcc
 	"
+AUDIO=wave LIB_PTHREAD= \
+VISIBILITY_FLAG= \
+PREFIX=$prefix \
+DATADIR=$dataDir/espeak-data \
+INCDIR=$includeDir/espeak
```

Comment:
Only the STDCPPLIBS change is required. You should also set STDCPPLIBS before using it.